content/docs/01-introduction.md

```diff
@@ -74,6 +81,10 @@ You can run `starship --help` to get a quick overview of the statusline function
 The final step to setting up your terminal statusline is to install a [NerdFont](https://www.nerdfonts.com/font-downloads). This step is technically optional, your statusline will function without it, but having a NerdFont installed will allow you to see helpful icons.
 Once you finish downloading and installing your NerdFont, use it by opening your iTerm preferences, going to the `Profile` section, and selecting the `Text` tab.
 
+If you get decision fatigue and cant decide which font to install, [this](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/IBMPlexMono.zip) is a fine choice.
```

Comment:
is there a `brew install` step we can do here?

Reply:
good call, added those instructions